src/Root.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { BrowserRouter, Route, Routes, Navigate } from 'react-router-dom';
+import { App } from './App';
+import { AccessoriesPage } from './pages/AccessoriesPage/AccessoriesPage';
+import { CartPage } from './pages/CartPage/CartPage';
+import { DetailsPage } from './pages/DetailsPage/DetailsPage';
+import { FavoritesPage } from './pages/FavoritesPage/FavoritesPage';
+import { HomePage } from './pages/HomePage/HomePage';
+import { PhonesPage } from './pages/PhonesPage/PhonesPage';
+import { TabletsPage } from './pages/TabletsPage/TabletsPage';
+
+export const Root: React.FC = () => (
+  <BrowserRouter>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to="/" replace />} />
+        <Route path="phones" element={<PhonesPage />} />
+        <Route path="tablets" element={<TabletsPage />} />
+        <Route path="accessories" element={<AccessoriesPage />} />
+        <Route path="phones/:productId" element={<DetailsPage />} />
+        <Route path="tablets/:productId" element={<DetailsPage />} />
+        <Route path="accessories/:productId" element={<DetailsPage />} />
+        <Route path="cart" element={<CartPage />} />
+        <Route path="favorites" element={<FavoritesPage />} />
+        {/* <Route path="*" element={<ErrorPage />} /> */}
+      </Route>
+    </Routes>
+  </BrowserRouter>
```

Comment:
```suggestion
  <HashRouter>
    <Routes>
        <Route index element={<HomePage />} />
         <Route path="phones">
                 <Route index element={<PhonesPage />} />
                 <Route path=":phoneId" element={<DetailsPage />} />
         </Route>
         <Route path="tablets">
                 <Route index element={<TabletsPage />} />
                 <Route path=":tabletId" element={<DetailsPage />} />
         </Route>
         <Route path="tablets">
                 <Route index element={<AccessoriesPage />} />
                 <Route path=":accsesoryId" element={<DetailsPage />} />
         </Route>
        <Route path="cart" element={<CartPage />} />
        <Route path="favorites" element={<FavoritesPage />} />
        <Route path="home" element={<Navigate to="/" replace />} />
        {/* <Route path="*" element={<ErrorPage />} /> */}
    </Routes>
  </HashRouter>
```

add some structure, so we can see where exactly pages leading 
also add a Hash router to deploy page on GitHub